apps/onesky-app/app/global.css

```
@@ -3,502 +3,506 @@
+
+.progress-bar {
+  border-radius: 9999;
+  background-color: #e5e5e5;
+  height: 7px;
+  width: 85%;
+  margin-top: 6px;
+}
+
+.activities-page {
+  padding-top: 0%;
+  padding: 5%;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)